library/core/src/num/nonzero.rs

```diff
@@ -287,6 +287,204 @@ nonzero_integers_div! {
     NonZeroUsize(usize);
 }
 
+macro_rules! nonzero_integers_add {
+    ( $( $Ty: ident($Int: ty); )+ ) => {
+        $(
+            impl $Ty {
+                /// Add a positive integer to a non-zero value.
```

Comment:
`positive integer` is often used to mean `> 0`, not `>= 0`. `nonnegative` or `unsigned` might be better.